lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -2462,6 +2462,23 @@ class DecomposeAten_EmbeddingBagOp
 };
 } // namespace
 
+// Decompose aten.div.int into aten.div.float
+namespace {
+class DecomposeAtenDivIntOp : public OpRewritePattern<AtenDivIntOp> {
```

Comment:
Decompositions should also have lit tests in https://github.com/llvm/torch-mlir/blob/main/test/Dialect/Torch/decompose-complex-ops.mlir